h2o-core/src/main/java/hex/grid/HyperParameters.java

```diff
@@ -0,0 +1,79 @@
+package hex.grid;
+
+import water.AutoBuffer;
+import water.H2O;
+import water.Iced;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * Wrapper class to make hyper-parameters serializable with Icer
+ */
+public class HyperParameters extends Iced<HyperParameters> {
+
+    private volatile Map<String, Object[]> values;
```

Comment:
volatile? Maybe transient instead? to make it clear it is not serialized?

Reply:
good catch, fixed